lib/releases.js

```diff
@@ -237,17 +237,36 @@ const generateChangeLog = (mergedPullRequests, config) => {
     changeLog.push(pullRequestToString(uncategorizedPullRequests), '\n\n')
   }
 
-  categorizedPullRequests.map((category, index) => {
-    if (category.pullRequests.length > 0) {
+  for (const [index, category] of categorizedPullRequests.entries()) {
+    if (category.pullRequests.length === 0) {
+      continue
+    }
+    const categoryTitle = template(config['category-template'], {
+      $TITLE: category.title,
+    })
+    const pullRequestString = pullRequestToString(category.pullRequests)
+
+    if (
+      !category.collapse ||
+      (category.collapse && category.pullRequests.length < 4)
+    ) {
+      changeLog.push(categoryTitle, '\n\n', pullRequestString)
+    } else {
       changeLog.push(
-        template(config['category-template'], { $TITLE: category.title }),
+        categoryTitle,
+        '\n',
```

Comment:
Heading should have double newline.
```suggestion
        '\n\n',
```